src/main/java/org/opensearch/security/securityconf/ConfigModelV7.java

```diff
@@ -563,6 +600,53 @@ private boolean containsDlsFlsConfig() {
         }
     }
 
+    public static class OnBehalfOfSettings {
```

Comment:
Should keep this part.

Reply:
@RyanL1997 this is required correct?

Reply:
We should keep this for sure, but I just checked the main branch this should be located in ConfigV7.java. Should we also move that?

Reply:
I have refactored it.